tests/lib/rules/no-useless-escape.js

```diff
@@ -72,8 +72,8 @@ ruleTester.run("no-useless-escape", rule, {
         {code: "var foo = `${foo}\\f`;", parserOptions: {ecmaVersion: 6}},
         {code: "var foo = `${foo}\\\n`;", parserOptions: {ecmaVersion: 6}},
         {code: "var foo = `${foo}\\\r\n`;", parserOptions: {ecmaVersion: 6}},
-        {code: "var foo = `\\\``", parserOptions: {ecmaVersion: 6}},
-        {code: "var foo = `\\\`${foo}\\\``", parserOptions: {ecmaVersion: 6}},
+        {code: "var foo = `\\\``", parserOptions: {ecmaVersion: 6}}, // eslint-disable-line no-useless-escape
```

Comment:
Not sure what the intent of this test is either, but again we should be able to resolve this as either two backslashes total or four.